migrations/20240723144352-add-comments-to-host-applications.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('Comments', 'HostApplicationId', {
+      type: Sequelize.INTEGER,
+      allowNull: true,
+      references: {
+        key: 'id',
+        model: 'HostApplications',
+      },
+    });
+
+    await queryInterface.addColumn('CommentHistories', 'HostApplicationId', {
+      type: Sequelize.INTEGER,
+      allowNull: true,
+    });
```

Comment:
Could be nice to add an index to avoid the increasing complexity as the table grows, similar to what we do for the other comment types:

```
    await queryInterface.addIndex('Comments', ['HostApplicationId'], {
      concurrently: true,
      where: {
        HostApplicationId: { [Op.ne]: null },
      },
    });
```